README.md

```diff
@@ -1,2 +1,122 @@
-# graphql-directive-uid
-@uid directive for GraphQL
+<div align="center">
+  <h1>@uid directive for GraphQL</h1>
+  <p>Directive for GraphQL schema.</p>
+</div>
+
+<hr />
+
+## The problem
+
+A lot of people have problem with generating unique `ID` for each graphql schema type
+
+## This solution
+
+The `@uid` is the simple graphql directive that allo user to have minimum effort but have all benefits from graphql cache.
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+* [Installation](#installation)
+* [Usage](#usage)
+* [More on `data-testid`s](#more-on-data-testids)
+* [Examples](#examples)
+* [Contributors](#contributors)
```

Comment:
Can we remove "Contributors" section? I love the idea, but it's a burden to maintain the list (e.g. asking everybody to run a script to include themselves on the list)